_includes/c2s_structure.md

```diff
@@ -68,7 +68,9 @@ For more information on processing tags – including the naming and registratio
 
 ### Source
 
-      source          ::=  servername / ( nickname [ "!" user ] [ "@" host ] )
+      source          ::=  <servername> / ( <nickname> [ "!" <user> ] [ "@" <host> ] )
+      nick            ::=  <any characters except NUL, CR, LF, #, chantype character, and SPACE> <possibly empty sequence of any characters except NUL, CR, LF, and SPACE>
```

Comment:
You have both # and "chantype character", can you clarify what that means?

Reply:
It means I didn't pay enough attention to what I wrote

Reply:
```suggestion
      nick            ::=  <any characters except NUL, CR, LF, chantype character, and SPACE> <possibly empty sequence of any characters except NUL, CR, LF, and SPACE>
```